packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -652,19 +653,30 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     // If autoComplete is on, attempt to find a match from the available options
     if (this.props.autoComplete === 'on') {
       // If autoComplete is on, attempt to find a match where the text of an option starts with the updated value
-      const items = currentOptions
-        .map((item, index) => {
-          return { ...item, index };
-        })
-        .filter(
-          option => option.itemType !== SelectableOptionMenuItemType.Header && option.itemType !== SelectableOptionMenuItemType.Divider
-        )
-        .filter(
+      let items = currentOptions.map((item, index) => {
+        return { ...item, index };
+      });
```

Comment:
Why is `processInputChangeWithoutFreeform` also being updated?

Reply:
This method is  _processInputChangeWithFreeform, not without freeform. There are no changes to that method in this PR.

Reply:
Updated my comment, there was a missing 'not'...

---
In reply to: [218909778](https://github.com/OfficeDev/office-ui-fabric-react/pull/6388#discussion_r218909778) [](ancestors = 218909778)